lib/proxy/plugin_initializer.rb

```diff
@@ -114,7 +114,7 @@ def configure
   end
 
   def fail_group(an_exception)
-    fail_group_with_message("Disabling all modules in the group #{printable_module_names(member_names)} due to a failure in one of them: #{an_exception}", an_exception.backtrace)
```

Comment:
Would you mind submitting this as a separate PR? Looks like a good fix, but an unrelated fix.

Reply:
Sure, created: https://projects.theforeman.org/issues/33482

Edit: Ooops this was fixed in b94c9b07